.github/workflows/ci.yaml

```
@@ -28,3 +28,10 @@ jobs:
         run: make generate && make checkgenerate
       - name: Build
         run: make build && make checkgenerate
+      - uses: bufbuild/buf-setup-action@v1.29.0-1
+      - run: |
```

```suggestion
      - uses: bufbuild/buf-setup-action@v1
      - run: |
```
nit: probably better give `name` to this 2, also maybe just use v1 so that it just use the latest release?

---

Added names.